staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -201,18 +204,71 @@ func (l FilteredLogger) log(skipFrames int, params ...interface{}) error {
 			"component", l.component,
 		)
 		if l.err != nil {
-			l.logger = log.With(l.logger, "reason", l.err)
+			l.logger = klog.With(l.logger, "reason", l.err)
 		}
-		return log.WithPrefix(l.logger, logParams...).Log(params...)
+		return klog.WithPrefix(l.logger, logParams...).Log(params...)
 	}
 	return nil
 }
 
+func (l FilteredLogger) Key(key string, kind string) *FilteredLogger {
+	if key == "" {
+		return &l
+
+	}
+	name, namespace, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		return &l
+	}
+	logParams := make([]interface{}, 0)
+	if namespace != "" {
+		logParams = append(logParams, "namespace", namespace)
+	}
+	logParams = append(logParams, "name", name)
+	logParams = append(logParams, "kind", kind)
+	l.with(logParams...)
+	return &l
+}
+
+func (l FilteredVerbosityLogger) Log(params ...interface{}) error {
+	return l.filteredLogger.log(2, params...)
+}
+
+func (l FilteredVerbosityLogger) V(level int) *FilteredVerbosityLogger {
+	if level >= 0 {
+		l.filteredLogger.currentVerbosityLevel = level
+	}
+	return &l
+}
+
+func (l FilteredVerbosityLogger) Info(msg string) {
+	l.filteredLogger.Level(INFO).log(2, "msg", msg)
```

Comment:
Is this hardcoding a verbosity of `2`?

Reply:
The `2` is for is `skipFrames` value for `runtime.Caller()` if I'm not mistaken.